src/playback/internal/playbackcontroller.h

```diff
@@ -125,7 +125,8 @@ class PlaybackController : public IPlaybackController, public muse::actions::Act
     muse::async::Channel<muse::actions::ActionCode> m_actionCheckedChanged;
 
     muse::async::Notification m_currentSequenceIdChanged;
-    playback::msecs_t m_currentPlaybackTimeMsecs = 0;
+    muse::secs_t m_currentPlaybackTimeMsecs = 0;
+    muse::secs_t m_lastPlaybackTimeSeekMsecs = 0;
```

Comment:
m_currentPlaybackTimeMsecs  -> m_currentPlaybackTime  
m_lastPlaybackTimeSeekMsecs -> m_lastPlaybackSeekTime

`M` - exactly what needs to be removed (in seconds, not milliseconds)
but I think `Sec` can also be removed, because now everything should be in `sec` by default, and the data type shows in what units of measurement.